SeaPkg/Tools/GenSeaArtifacts/gen_aux/src/auxgen.rs

```diff
@@ -171,18 +178,53 @@ impl <'a> ctx::TryIntoCtx<Endian> for &ImageValidationEntryHeader {
 }
 
 impl ImageValidationEntryHeader {
-    /// Generates an ImageValidationEntryHeader from a ValidationRule and a Symbol.
-    fn from_rule(rule: &ValidationRule, symbol: &Symbol) -> Self {
-        let mut entry = ImageValidationEntryHeader::default();
-        entry.offset = ((symbol.address as i64) + rule.offset.unwrap_or_default()) as u32;
-        entry.size = rule.size.unwrap_or(symbol.size);
-        entry.validation_type = rule.validation.clone();
-        if let Some(ref field) = rule.field {
-            entry.symbol = format!("{}.{}", &rule.symbol, field);
-        } else {
-            entry.symbol = rule.symbol.clone();
+    /// Generates one or more ImageValidationEntryHeaders from a ValidationRule and a Symbol.
+    ///
+    /// Multiple headers are generated if the top level symbol is a list of an underlying type.
+    /// In this scenario, the same rule is applied to each element in the list rather than the
+    /// list as a whole.

```

Comment:
Similar to any earlier comment in the PR, I'd drop "list" here and replace it either with "array" (preferred since this is C and that's clear per the static arrangement of contents in memory - fixed size and random access) or "sequenced type" (less preferred since it's unnecessarily ambiguous). "list" will be associated with linked lists for a lot of people in C.

Reply:
This makes sense. Updated all references to list to be "array"